projects/github-actions/repo-gardening/src/tasks/add-labels/index.js

```diff
@@ -336,20 +337,45 @@ async function addLabels( payload, octokit ) {
 	// If the PR title includes the word "revert", mark it as such.
 	const isRevert = title.toLowerCase().includes( 'revert' );
 
-	const labels = await getLabelsToAdd( octokit, owner.login, name, number, isDraft, isRevert );
+	const currentLabels = payload.pull_request.labels.map( l => l.name );
+	let labelsToAdd = await getLabelsToAdd(
+		octokit,
+		owner.login,
+		name,
+		number,
+		isDraft,
+		isRevert,
+		currentLabels
+	);
 
-	if ( ! labels.length ) {
-		debug( 'add-labels: Could not find labels to add to that PR. Aborting' );
+	// Nothing new was added, so abort.
+	if ( labelsToAdd.length === currentLabels.length ) {
+		debug( 'add-labels: No new labels to add to that PR. Aborting.' );
 		return;
 	}
 
-	debug( `add-labels: Adding labels ${ labels } to PR #${ number }` );
+	// GitHub allows 100 labels on a PR. Limit to less than that to allow for additional labels elsewhere.
+	const maxLabels = 90;
+	const bigLabel = 'All the things (>' + maxLabels + ' labels)';
```

Comment:
That's just a personal preference, but I find template literals a lot easier to read.

```suggestion
	const bigLabel = `All the things (> ${ maxLabels } labels)`;
```

Reply:
9661caa15c